pkgs/development/compilers/ghc/8.2.1.nix

```diff
@@ -1,29 +1,58 @@
-{ stdenv, lib, fetchurl, bootPkgs, perl, ncurses, libiconv, binutils, coreutils
+{ stdenv, fetchurl, bootPkgs, perl, binutils, coreutils
 , autoconf, automake, happy, alex, python3, sphinx, hscolour
-, buildPlatform, targetPlatform , selfPkgs, cross ? null
-
-  # If enabled GHC will be build with the GPL-free but slower integer-simple
+, ncurses, gmp, libffi
+, __targetPackages
```

Comment:
Can you document this and other non-package non-obvious arguments?

Reply:
Now documented in manual